kubernetes/assets/dashboards/kubernetes_pods.json

```diff
@@ -147,7 +147,7 @@
                 "type": "timeseries",
                 "requests": [
                     {
-                        "q": "sum:kubernetes_state.container.ready{$cluster,$namespace,$deployment,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.container.running{$cluster,$namespace,$deployment,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.container.terminated{$cluster,$namespace,$deployment,$scope,$statefulset,$daemonset,$job}, sum:kubernetes_state.container.waiting{$cluster,$namespace,$deployment,$scope,$statefulset,$daemonset,$job}",
+                        "q": "sum:kubernetes_state.container.ready{$kube_cluster_name,$kube_namespace,$kube_deployment,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.container.running{$kube_cluster_name,$kube_namespace,$kube_deployment,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.container.terminated{$kube_cluster_name,$kube_namespace,$kube_deployment,$scope,$kube_stateful_set,$kube_daemon_set,$job}, sum:kubernetes_state.container.waiting{$kube_cluster_name,$kube_namespace,$kube_deployment,$scope,$kube_stateful_set,$kube_daemon_set,$job}",
```

Comment:
same for this metric, the only relevant tags we have are kube_cluster_name, kube_namespace, and kube_container_name: https://app.datadoghq.com/metric/summary?filter=kubernetes_state.container.ready&metric=kubernetes_state.container.ready